woocommerce/woocommerce
# Update TS config to have an isolated TS environment for woo admin

### All Submissions:






### Changes proposed in this Pull Request:



When we run `pnpm nx ts:check woocommerce-admin,` it also checks the woo packages. But everything in the monorepo is supposed to be isolated. So I made the following changes to have an isolated environment:

- Change the admin TS config and removed client TS config 
- Update the packages TS config to output declaration files 
- Move `plugins/woocommerce-admin/typings` to `plugins/woocommerce-admin/client/typings` to load them correctly.
- Disable `ForkTsCheckerWebpackPlugin` for now

### How to test the changes in this Pull Request:

1. Run `pnpm nx build woocommerce-admin` to build packages
2. `packages/js/data/build-types` should be generated
3. Run `pnpm nx ts:check woocommerce-admin`
4. Should see many client TS errors.
5. Run `pnpm nx clean woocommerce-admin & pnpm nx ts:check woocommerce-admin`
6. Should see some TS errors like `Cannot find module '@woocommerce/data' or its corresponding type declarations.` 

### Other information:




### FOR PR REVIEWER ONLY:
